llvm/include/llvm/ProfileData/InstrProfReader.h

```diff
@@ -377,16 +376,7 @@ class RawInstrProfReader : public InstrProfReader {
         NamesStart(nullptr), NamesEnd(nullptr), ValueDataStart(nullptr),
         ValueKindLast(0), CurValueDataSize(0), BinaryIds({}), Warn(Warn) {}
 #else
-  RawInstrProfReader(std::unique_ptr<MemoryBuffer> DataBuffer,
-                     const InstrProfCorrelator *Correlator,
-                     std::function<void(Error)> Warn)
-      : DataBuffer(std::move(DataBuffer)),
-        Correlator(dyn_cast_or_null<const InstrProfCorrelatorImpl<IntPtrT>>(
-            Correlator)),
-        Warn(Warn) {}
-#endif // SYCLomatic_CUSTOMIZATION
-=======
-  RawInstrProfReader(
+    RawInstrProfReader(
       std::unique_ptr<MemoryBuffer> DataBuffer,
       const InstrProfCorrelator *Correlator,
```

Comment:
Should align the params to upstream

Reply:
Double confirm, the code already followed the open source params.